serviceconnector/src/createServiceConnector/TargetServiceTypeStep.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { INewStorageAccountDefaults, StorageAccountKind, StorageAccountListStep, StorageAccountPerformance, StorageAccountReplication } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { TargetServiceType, TargetServiceTypeName } from "../../constants";
+import { ICreateLinkerContext } from "./ICreateLinkerContext";
+
+export class TargetServiceTypeStep extends AzureWizardPromptStep<ICreateLinkerContext>{
+    public async prompt(context: ICreateLinkerContext): Promise<void> {
+        const targetServiceLabels: string[] = [
+            vscode.l10n.t('Azure Storage - Blob'),
+            vscode.l10n.t('Azure Storage - Queue'),
+            vscode.l10n.t('Azure Storage - Table'),
+            vscode.l10n.t('Azure Storage - File'),
+        ];
+
+        const placeHolder = vscode.l10n.t('Select Target Service Type');
+        const picks: IAzureQuickPickItem<TargetServiceType>[] = [
+            { label: targetServiceLabels[0], data: { name: "storageBlob", type: TargetServiceTypeName.storage, id: '/blobServices/default' } },
+            { label: targetServiceLabels[1], data: { name: "storageQueue", type: TargetServiceTypeName.storage, id: '/queueServices/default' } },
+            { label: targetServiceLabels[2], data: { name: "storageTable", type: TargetServiceTypeName.storage, id: '/tableServices/default' } },
+            { label: targetServiceLabels[3], data: { name: "storageFile", type: TargetServiceTypeName.storage, id: '/fileServices/default' } },
```

Comment:
Nit, but the other enums like `StorageAccountKind` use properties that start with a capital letter, so I think it would look better to do the same, i.e. `TargetServiceTypeName.Storage`